src/main/java/com/epam/lowcost/model/Flight.java

```diff
@@ -0,0 +1,80 @@
+package com.epam.lowcost.model;
+
+import java.math.BigInteger;
+import java.util.Date;
+import java.util.Objects;
+
+public class Flight {
```

Comment:
Please, investigate into Lombok. It helps you avoid setters, getters, default constructor...

Reply:
Added Lombok and changed code. Got rid of all getters/setter etc.